veterans/app/api/auth/auth.config.test.ts

```
@@ -0,0 +1,180 @@
+    });
+
+    it('should use test values in test environment', () => {
+      const googleProvider = authConfig.providers[0] as OAuthConfig<Profile>;
+      expect(googleProvider.options?.clientId).toBe('test-value');
+      expect(googleProvider.options?.clientSecret).toBe('test-value');
+      expect(authConfig.secret).toBe('test-secret');
+    });
+
+    it('should throw error for missing environment variables in non-test environment', () => {
+      const jestGlobal = global.jest;
+      delete (global as { jest: unknown }).jest;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)